packages/manager/apps/telecom/src/app/telecom/telephony/line/assist/rma/rma.controller.js

```diff
@@ -4,11 +4,18 @@ import { TEXT_FOR_MODAL, ACTION_TYPE, RMA_NEW_TYPE } from './rma.constants';
 
 export default class TelecomTelephonyLineAssistRmaCtrl {
   /* @ngInject */
-  constructor($translate, lineAssistRmaService, TucToast, TucToastError) {
+  constructor(
+    $translate,
+    lineAssistRmaService,
+    TucToast,
+    TucToastError,
+    SUPPORT_URL,
```

Comment:
Isn't this missing an angularjs constant declaration ?
I don't see any, and as a result I don't see how this injection is supposed to work